<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAccount</class>
 <widget class="QDialog" name="MyAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/account.png</normaloff>:/img/account.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
border: solid 0px #ffffff;
background-color: rgb(9, 100, 186);
padding-top: 6px;
padding-bottom: 6px;
padding-left: 15px;
padding-right: 15px;
border-radius:18px;
color:#fff;
}

QPushButton:hover{
border: solid 2px rgb(0, 0, 0);
background-color: rgb(7, 78, 145);
padding-top: 6px;
padding-bottom: 6px;
padding-left: 15px;
padding-right: 15px;
border-radius:18px;
}

QDialog{
	background-color:rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="gridLayout_3">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:rgb(40, 40, 40);
	border-radius:15px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="11" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="18" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="0">
       <widget class="QSpinBox" name="balanceSb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="usernameLb">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Your username</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="nameSurnameLb">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Your name and surname</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Name  Surname</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="phoneNumLb">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Your phone number</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="20" column="0">
       <widget class="QPushButton" name="logoutPb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold SemiConden</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:34px;</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="23" column="0">
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="19" column="0">
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QPushButton" name="replenishBalancePb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold SemiConden</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:54px;</string>
        </property>
        <property name="text">
         <string>Replenish</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="changePasswordPb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold SemiConden</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:10px;
margin-right:10px;</string>
        </property>
        <property name="text">
         <string>Change password</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="balanceLb">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold SemiConden</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>       Replenish the balance(₴)       </string>
        </property>
       </widget>
      </item>
      <item row="24" column="0">
       <widget class="QPushButton" name="deleteAccountPb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold SemiConden</family>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:10px;
margin-right:10px;</string>
        </property>
        <property name="text">
         <string>Delete account</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
	color:rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>My account</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
